### ISSUE TITLE
Outlook Desktop, keyboard focus is lost after replying to an email with a pinned add-in

### ISSUE THREAD

**Issue by DmitryKostochko** [OfficeDev/office-js#2369]
We are developing an Outlook add-in for compose mode that supports the Pin/Unpin functionality. In Outlook Desktop, the keyboard focus is lost after replying to an existing email if an add-in is pinned.

## Your Environment
* Platform: PC desktop
* Host: Outlook
* Office version number: Version 2112 (Build 14729.20194)
* Operating System: Windows 10 Version 21H1 (19043.1415)
* Browser (if using Office on the web): 
 
## Expected behavior
The keyboard focus shall remain in the Body field in case of Reply/ReplyAll/Forward.

## Current behavior
The keyboard focus is lost, the user needs to click in the Body field to start typing.

## Steps to reproduce
1. In Outlook Desktop, select an email in your Inbox, click Reply, start the add-in and pin it.
2. **Close Outlook and open it anew.**
3. Select an email in your Inbox, click Reply or Reply All or Forward.
4. Note, the keyboard focus is lost, and you cannot type in the Body field.

## Link to live example(s)
You can use our test add-in:
https://online-addins.ablebits.com/testaddin/manifest.xml
Or any other add-in for compose mode that supports pin/unpin.

**Comment by exextoc** [comment]
I was able to repro this only on reboot of Outlook. Once an add-in has been pinned. If you skip the second step of your repro everything is fine? 

In other words, if I just pin an add-in to a compose form, then reply to an email. It's fine?

Internal Tracking Id: Office: 5728681

**Comment by DmitryKostochko** [comment]
In general, you are right. To be more precise, the issue occurs just once - when replying to the first message after (re)starting Outlook.

Here's a typical use case:
* The user pins the add-in to a compose form, then replies to an email - all is fine.
* When all the work is done, the user closes Outlook.
* Next day, the user opens Outlook and is unable to reply to the first message. Very annoying...

**Comment by exextoc** [comment]
Thanks for the report, we've opened a bug with the core team about this, I don't have a timetable to share on when it will be resolved.

**Comment by exextoc** [comment]
For tracking purposes, the product bug ID = 5728681.

**Comment by exextoc** [comment]
@DmitryKostochko, thanks for your patience. We have fixed this issue in build 16.0.15028.20000. This build is still rolling out, so expect the fix in an update within the next few weeks.
